src/rotary-dailer-element.stories.ts

```diff
@@ -7,4 +7,6 @@ import { logEvent } from 'storybook-events-logger';
 storiesOf('Rotary Dialer', module)
   .addParameters({ component: 'wokwi-rotary-dialer' })
   .addDecorator(withKnobs)
-  .add('Default', () => html` <wokwi-rotary-dialer @dial=${logEvent}></wokwi-rotary-dialer> `);
+  .add('Start', () => html` <wokwi-rotary-dialer @dial-start=${logEvent}></wokwi-rotary-dialer> `)
```

Comment:
Why not keep a single story with all three events?
This way it's easier to see the temporal relation between the events (i.e. their order).

Usually different stories are used to display different visual style options of the element

Reply:
noted! I'lll change